docs/indexer/quickstart/how-it-works.md

```diff
@@ -0,0 +1,160 @@
+# 2. How it works
+
+The SubQuery CLI creates a working template project as a launch pad for your own custom project. From this template, there are only 3 important files you need to familiarise yourself with.
+These are:
+
+1. [The project.ts file](#the-projectts-file)
+2. [The GraphQL Schema file](#the-graphql-schema-file)
+3. [The Mapping functions (in src/mappings/ directory)](#the-mapping-functions)
+
+## The project.ts file
+
+The project.ts file can be seen as an entry point of your project as it defines most of the details on how SubQuery will index and transform the chain data. It indicates where we are indexing data from, and to what on chain events we are subscribing to.
+
+This file imports various libraries and configurations along with defining the name, version and description of the project. In the network section, it specifies the chainID, the RPC endpoint, and also the startBlock. It also specifies the ABI contract address.
```

Comment:
Generally there shouldn't be any library imports here, just a type definition and environment variables.

Reply:
Updated.